### PR TITLE
feat: updates to download resources page

### PR DESCRIPTION
**Pull request by mantagen**
## Description

- Changes to 'download' page as [specified in Miro](https://miro.com/app/board/uXjVKI-2qEQ=/)
- Change progress to show 10 sections rather than 12 (reducing the first three into `lessonDetails`)
- Copy updates
- **Important:** changes functionality of progress checker to reflect the reality that _1 learning cycle_ is a valid state (so 'Learning cycles' gets a tick in this case)
- Changes a bunch of copy to use **Sentence case**

## Issue(s)

Fixes https://www.notion.so/oaknationalacademy/Update-descriptions-of-each-section-on-the-download-resources-page-164770c7a7fa419c94a8367cf64f3d8f?pvs=4

## How to test

1. Go to https://oak-ai-lesson-assistant-a59flyrtc.vercel-preview.thenational.academy
2. Go to a chat -> downloads
3. You should see new progress sections as in the image below
4. You should see the 10 sections change reflected in the progress bar on the chat too

## Screenshots

How it used to look (delete if n/a):

How it should now look:

image

### CODE REVIEW

**Review by stefl — APPROVED** [review]
Looks good - glad we're now using the same completeness code in both places

File: apps/nextjs/src/app/aila/[id]/download/DownloadView.tsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    lessonSlidesExport,
+    worksheetExport,
+    lessonPlanExport,
+    starterQuizExport,
+    exitQuizExport,
+    additionalMaterialsExport,
+    sections,
+    totalSections,
+    totalSectionsComplete,
+  } = useDownloadView(chat);
```
Comment (mantagen) [inline apps/nextjs/src/app/aila/[id]/download/DownloadView.tsx:35]: This diff is mostly changing the file name. But the introduction of this hook is the substantive change in this file

File: apps/nextjs/src/app/aila/[id]/download/DownloadView.tsx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+                      />
+                    )}
+                </Flex>
+                <Box>
+                  <div className="mb-17">
+                    <span className=" font-bold">
+                      {`${totalSectionsComplete} of ${totalSections} sections complete`}
+                    </span>
+                  </div>
+                  {sections.map((section) => {
```
Comment (mantagen) [inline apps/nextjs/src/app/aila/[id]/download/DownloadView.tsx:151]: Also using this sections array rather than what was in the previous version

**Comment by mikeritson-oak** [comment]
General changes and functionality of this fix are looking good. One thing I noted was that there was a request in Miro for a couple of icon changes but these don't seem to have been reflected in this PR:
image

**Pull request merged**